ModelCatalogueGenomicsPlugin/grails-app/services/modelcataloguegenomicsplugin/GelJsonService.groovy

```diff
@@ -0,0 +1,160 @@
+package modelcataloguegenomicsplugin
+
+
+import grails.transaction.Transactional
+import org.modelcatalogue.core.CatalogueElement
+import org.modelcatalogue.core.DataClass
+
+
+//@Transactional
+class GelJsonService {
```

Comment:
on the other hand this solution when done right (see 6c9408c91c2770bc87f58c54a1da6849eee67598) is incredibly fast so let's keep it as it is